06_neighborhoods/Transportation/transit_trips_cost_city.qmd

```diff
@@ -371,13 +505,17 @@ Examine outliers
 ggplot(transit_cost_city_2015, aes(x=index_transportation_cost)) + geom_histogram(binwidth=10) + labs(y="number of places", x="Annual Transit Cost for the Regional Moderate Income Household, 2015")
 
 ggplot(transit_cost_city_2019, aes(x=index_transportation_cost)) + geom_histogram(binwidth=10) + labs(y="number of places", x="Annual Transit Cost for the Regional Moderate Income Household, 2019")
+
+ggplot(transit_cost_city_2020, aes(x=index_transportation_cost)) + geom_histogram(binwidth=10) + labs(y="number of places", x="Annual Transit Cost for the Regional Moderate Income Household, 2020")
```

Comment:
The way these ggplot visuals are turning out currently does not offer much information, I think the bin widths are set too wide - it is just appearing as one large grey square.